habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -253,26 +254,43 @@ def reset(self):
         return None
 
     def reconfigure(self, config: "DictConfig", ep_info: RearrangeEpisode):
-        self._handle_to_goal_name = ep_info.info["object_labels"]
-
         t_start = time.time()
 
-        with read_write(config):
-            config["scene"] = ep_info.scene_id
-
         self.ep_info = ep_info
         new_scene = self.prev_scene_id != ep_info.scene_id
         if new_scene:
             self._prev_obj_names = None
 
+        if self._force_soft_reset:
```

Comment:
Are these two objects also the target objects to rearrange?

Reply:
Yes, I added a comment to address these as well.